packages/styles/components/Alert.module.scss

```diff
@@ -3,28 +3,103 @@
 $colors: 'primary' var(--primary-color), 'secondary' var(--secondary-color),
   'success' var(--success-color), 'danger' var(--danger-color);
 
-$iconSize: var(--space-sm);
+$icon-sz-sm: 12px;
+$icon-sz: var(--space-sm);
+$icon-sz-lg: 24px;
```

Comment:
I took a delay to understand what `sz` means, maybe is better use a explicit name like `size`.

Reply:
made on 660a5446171c07d6d9ca3645380979bdd937afcf